### PR TITLE
feat: failed to deliver

### PR DESCRIPTION
**Pull request by Graborg**
This PR adds the functionality for the user to mark the package as "delivery_failed"

### CODE REVIEW

File: packages/driver-interface/services/messaging.js [hunk 1]
```diff
@@ -170,7 +170,9 @@ const sendDeliveryInformation = (instruction, telegramId, booking) =>
           ? `\nPortkod: ${booking.metadata.recipient.doorCode}`
           : ''
       )
-      .concat(`\nTryck "[Levererat]" när du har lämnat paketet.`),
+      .concat(
+        `\nTryck "[Levererat]" när du har lämnat paketet, eller "Kunde inte leverera" om du av någon anledninge inte kunde leverera paketet.`
```
Comment (lundeeell) [inline packages/driver-interface/services/messaging.js:174]:
```suggestion
        `\nTryck "[Levererat]" när du har lämnat paketet, eller "[Kunde inte leverera]" om du av någon anledning inte kunde leverera paketet.`
```

File: packages/driver-interface/botRoutes.js [hunk 2]
```diff
@@ -122,14 +93,16 @@ const init = (bot) => {
...
         return onArrived(msg)
+      case 'picked_up':
       case 'delivered':
-        return onDelivered(msg)
-      case 'offer':
-        return onOffer(msg)
+      case 'delivery_failed':
+        const { id: telegramId } = msg.update.callback_query.from
+        const { e: event, id: bookingId } = callbackPayload
+        return handleBookingEvent(telegramId, bookingId, event)
```
Comment (lundeeell) [inline packages/driver-interface/botRoutes.js:105]:
```suggestion
      case 'delivery_failed': {
        const { id: telegramId } = msg.update.callback_query.from
        const { e: event, id: bookingId } = callbackPayload
        return handleBookingEvent(telegramId, bookingId, event)
      }
```

Comment (lundeeell) [reply packages/driver-interface/botRoutes.js:105]: Lexical declaration

**Review by lundeeell — CHANGES_REQUESTED** [review]
Booking is not updated in UI. engine-server is not listening to new key "delivery_failed"

File: packages/driver-interface/services/messaging.js (hunk 1 above)
Comment (Graborg) [reply packages/driver-interface/services/messaging.js:174]: 🙏

**Review by lundeeell — APPROVED** [review]

**Pull request merged**